pkg/app/piped/imagewatcher/watcher.go

```diff
@@ -143,7 +143,7 @@ func (w *watcher) run(ctx context.Context, repo git.Repo, repoCfg *config.PipedR
 				continue
 			}
 			if !ok {
-				w.logger.Error("configuration file for Image Watcher not found",
+				w.logger.Info("configuration file for Image Watcher not found",
```

Comment:
nits, I don't get the point of this change 🤔

Reply:
These workers start running even if a piped operator doesn't intend to enable Image Watcher. So I'd say what it's not found isn't an error.

Reply:
ah, you're right, thanks 👍